### ISSUE TITLE
Scroll not recovering after deletion

### ISSUE THREAD

**Issue by Shatur** [dstein64/nvim-scrollview#70]
**Description**

Some plugins may remove buffers, for example https://github.com/Shatur/neovim-session-manager
But scrollview scroll not recovering after deletion.

**Steps to reproduce**

Run the following code:

```lua
for _, buffer in ipairs(vim.api.nvim_list_bufs()) do
    vim.api.nvim_buf_delete(buffer, {})
end
```

**Current behavior**

Scroll no longer appears after opening new buffers.

**Expected behavior**

Scroll appears on opening new buffers.

**Comment by dstein64** [comment]
Thanks for reporting this @Shatur!

The issue arose when I ported the plugin to Lua in PR #47, as I mistakenly didn't convert a VimScript boolean to Lua. This is fixed in ba6c48bf7919dd48a371e7b9c683b8858053e885.

**Comment by Shatur** [comment]
Thanks!
